packages/design-system/changelog/unreleased/enhancement-condensed-menu-item

```diff
@@ -0,0 +1,5 @@
+Enhancement: Add condensed menu icon 
```

Comment:
@kulmann not sure if the changelog in the design system still gets updated individually or if there's only one in the repo root from now on?

Reply:
We need to maintain a separate changelog, because we want to be able to release independently with non aligned version numbers

Reply:
Good, that's what I assumed (and hence the changelog entry^^)

Reply:
yes, this ^